src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -81,8 +81,48 @@ export const addEmailAddressCreateTransactionFailure = [
   }),
 ];
 
+// When the initial request to update an email address fails
+export const updateEmailAddressCreateTransactionFailure = [
+  rest.put(`${prefix}/v0/profile/email_addresses`, (req, res, ctx) => {
+    return res(
+      ctx.status(400),
+      ctx.json({
+        // response pulled from vets-api config/locales/exceptions.en.yml
+        errors: [
+          {
```

Comment:
this error object is repeated a few times in these mocks but I'm just going to leave it that way for now.